tools/generate.sh

```diff
@@ -14,4 +14,6 @@ function api {
 }
 
 api $DIR/api/migraphx.h $SRC_DIR/api/include/migraphx/migraphx.h
+echo "Finished generating header migraphx.h"
```

Comment:
Would it be possible to add comment strings inside `migraphx.h` that it is auto-generated, and shouldn't be changed?